webpack/assets/javascripts/react_app/components/hosts/storage/vmware/StorageContainer.test.js

```diff
@@ -27,7 +32,7 @@ describe('StorageContainer', () => {
       wrapper.render().find('.disk-container').length
     ).toEqual(1);
 
-    wrapper.find('.btn-add-disk').simulate('click');
+    wrapper.find('.btn-add-disk').at(0).at(0).simulate('click');
```

Comment:
why do these change? is it a wrong selector?

Reply:
Yeah, the old api only returned [one result](http://airbnb.io/enzyme/docs/guides/migration-from-2-to-3.html#element-referential-identity-is-no-longer-preserved) when a `.find` matched multiple elements. `wrapper.find('.btn-add-disk')` now returns both elements that match:

```jsx
    <Button className="btn-add-disk" disabled={false} onClick={[Function]} active={false} block={false} bsStyle="default" bsClass="btn">
      <button disabled={false} onClick={[Function]} type="button" className="btn-add-disk btn btn-default">
        Add volume
      </button>
    </Button>
    
    
    <button disabled={false} onClick={[Function]} type="button" className="btn-add-disk btn btn-default">
      Add volume
    </button>
```

I'm not sure if the outer wrapper and the inner button are both supposed to have the `btn-add-disk` class. If so maybe we could use a different selector. Thoughts?

Reply:
I changed the selector to `button.btn-add-disk` and that seems to do the trick.